cpp/bench/ann/CMakeLists.txt

```diff
@@ -15,9 +15,18 @@
 # ##################################################################################################
 # * benchmark options ------------------------------------------------------------------------------
 
-option(RAFT_ANN_BENCH_USE_FAISS_BFKNN "Include faiss' brute-force knn algorithm in benchmark" ON)
-option(RAFT_ANN_BENCH_USE_FAISS_IVF_FLAT "Include faiss' ivf flat algorithm in benchmark" ON)
-option(RAFT_ANN_BENCH_USE_FAISS_IVF_PQ "Include faiss' ivf pq algorithm in benchmark" ON)
+option(RAFT_ANN_BENCH_USE_FAISS_GPU_FLAT "Include faiss' brute-force knn algorithm in benchmark" ON)
+option(RAFT_ANN_BENCH_USE_FAISS_GPU_IVF_FLAT "Include faiss' ivf flat algorithm in benchmark" ON)
+option(RAFT_ANN_BENCH_USE_FAISS_GPU_IVF_PQ "Include faiss' ivf pq algorithm in benchmark" ON)
```

Comment:
I think some of our benchmarks also have `faiss_gpu_ivf_sq` algorithm, can you add a target for that as well? Right now it works because we build all faiss algorithms into all faiss named executables https://github.com/rapidsai/raft/blob/2f08bdd75286688161ca1c8ff3b608985be117bf/cpp/bench/ann/src/faiss/faiss_benchmark.cu#L123

Reply:
Also, below we have a CPU IVF SQ benchmark too, please add that target

Reply:
I'd prefer not to expose ivf sq at the moment. It's been in the code since the first iteration of the benchmarks and so was copied to the cpu version as well. It's thus far untested and not something we need to benchmark at the moment anyways.

Reply:
Okay, that sounds good to me!

Reply:
That being said, I'm going to create a GitHub issue to track the ivf sq and reference it in the code so we don't lose sight of it